home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
```

Comment:
- [ ] I think it would be great if you decrease the font sizes here for the texts since I can see the texts is too bog so please decrease the font sizes here as much as possible 😊💯✨

- [ ] I think it would be great if you also decrease the height and the width for the logo icon here since it is too big in your design ✨💯
- Your design
![image](https://user-images.githubusercontent.com/58553711/219922029-92436049-246c-4b01-a7b9-aafb8419bf02.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/219922160-92c4319f-26bb-49ac-90e3-a861079f5b1f.png)